pkg/status/render.go

```diff
@@ -146,9 +141,10 @@ func FormatDCAStatus(data []byte) (string, error) {
 	}
 
 	forwarderStats := stats["forwarderStats"]
-	runnerStats := stats["runnerStats"]
-	autoConfigStats := stats["autoConfigStats"]
-	checkSchedulerStats := stats["checkSchedulerStats"]
+	// We nil these keys because we do not want to display that information in the collector template
+	stats["pyLoaderStats"] = nil
+	stats["pythonInit"] = nil
+	stats["inventories"] = nil
```

Comment:
Could you create a task in our backlog to split the collector template so the DCA can use either it's own version of a section of `collector.tmpl` and we no longer need to nil some field.